compiler/rustc_lint/src/lib.rs

```diff
@@ -579,25 +580,25 @@ fn register_builtins(store: &mut LintStore) {
 }
 
 fn register_internals(store: &mut LintStore) {
-    store.register_lints(&LintPassImpl::get_lints());
+    store.register_lints(&LintPassImpl::default().get_lints());
```

Comment:
```suggestion
    store.register_lints(&LintPassImpl::get_lints());
```
and everywhere below.